### DISCUSSION TITLE
Combobox takes up all height of window when too many options

### DISCUSSION THREAD

**Discussion by jh-jensen** [TomSchimansky/CustomTkinter#1200] [Q&A]
Hello, is there any way to set a specific max height for the dropdown list of the Combobox widget? And with a scrollbar, i.e. the same behaviour as the ttk Combobox? I have a list of years going from 2023 all the way back to 1900-something, but when opening the Combobox dropdown, the dropdown takes up the entire computer screen (as in, even beyond the entire app window) which is not ideal.
Setting the height attribute just creates a white rectangle of some sort, not sure why?
[https://tinypic.host/i/capture.lohCj](url)

**Comment by CheetahDoesStuff** [comment]
I have the same issue, i havent found a fix sadly
